logstash.go

```diff
@@ -19,9 +22,35 @@ type LogstashAdapter struct {
 	route *router.Route
 }
 
+func getopt(name, dfault string) string {
+	value := os.Getenv(name)
+	if value == "" {
+		return dfault
+	}
+	return value
+}
+
+func strToSlice(str, delimiter string) []string {
+	var sliceStr []string
+	matched, _ := regexp.MatchString(delimiter, str)
+	if matched == true {
+		sliceStr = strings.Split(str, delimiter)
+	} else {
+		sliceStr = []string{str}
+	}
+	return sliceStr
+}
+
+func debug(v ...interface{}) {
+	if os.Getenv("DEBUG") != "" {
```

Comment:
I just realized that it might not me very efficient to call `os.Getenv()` multiple times for every log line. The best thing would be if you removed all debug calls and the debug method from this PR so we can merge without it. If you want you can create a new PR with the debugging. If you do that I it would be better to call `os.Getenv()` once in `NewLogstashAdapter` and set a debug flag in the `LogstashAdapter`. A test case with a performance benchmark for debug logging would be good to add in that case.